plugins/Diagnostics/Diagnostic/DiagnosticResult.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+/**
+ * The result of a diagnostic.
```

Comment:
But this is just a temporary hack, hopefully we'll make that cleaner at some point. If we document it, it becomes official and we need to support it.